### ISSUE TITLE
Northwestern Theme - Link to NU Press in footer has Insufficient Color Contrast

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#619]
FOOTER LINK: Element has insufficient color contrast of 4.40 (foreground color: #ffffff, background color: #836eaa, font size: 10.5pt, font weight: normal)

`.js.wf-droidserif-n4-active.wf-droidserif-i4-active.wf-freightsanspro-n4-active.wf-droidserif-i7-active.wf-freightsanspro-n3-active.wf-freightsanspro-i4-active.wf-droidserif-n7-active.wf-freightsanspro-n6-active.wf-freightsanspro-i6-active.wf-active > .northwestern > footer > .container-fluid > .row.col-wrap > .col-sm-4.press-block-a.col > p > .brand`
`<a class="brand" href="http://nupress.northwestern.edu/">Northwestern University Press</a>`

- [ ] 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.
